packages/common/src/ui/components/inputs/Autocomplete/Autocomplete.stories.tsx

```diff
@@ -18,66 +18,86 @@ const StyledPaper = styled(Paper)({
 });
 
 const options = Array.from({ length: 100 }).map((_, i) => ({
+  id: `${i}`,
   label: `${i}`,
 }));
 
 const longOptions = [
   {
     label: 'SAINT JOSEPH MOSCATI (HÔPITAL CATHOLIQUE)',
+    id: '1',
```

Comment:
I ❤️  GitHub copilot 😁

Reply:
Ooh - you'll have to show me that at some point